frontend/src/queries/schema.ts

```diff
@@ -240,6 +241,7 @@ export interface DataTableNode extends Node {
     hiddenColumns?: HogQLExpression[]
     /** Show with most visual options enabled. Used in scenes. */
     full?: boolean
+
```

Comment:
Gonna block everything for this space now. :police_hog:

Reply:
I'd guess the loading weirdness is some inconsistency with coordinating filters and queries state which is getting a bit tangly now. But will take a look...